package.json

```diff
@@ -57,79 +57,80 @@
-    "envalid": "7.0.0",
-    "eslint": "7.20.0",
-    "eslint-config-prettier": "8.0.0",
+    "envalid": "7.1.0",
+    "eslint": "7.24.0",
+    "eslint-config-prettier": "8.2.0",
     "eslint-plugin-jsx-a11y": "6.4.1",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-notice": "0.9.10",
-    "eslint-plugin-react": "7.23.0",
+    "eslint-plugin-react": "7.23.2",
     "eslint-plugin-react-hooks": "4.2.0",
-    "gatsby": "2.32.4",
+    "gatsby": "2.32.12",
     "gatsby-image": "2.11.0",
     "gatsby-node-helpers": "0.3.0",
     "gatsby-plugin-google-gtag": "2.8.0",
-    "gatsby-plugin-manifest": "2.12.0",
-    "gatsby-plugin-mdx": "1.10.0",
-    "gatsby-plugin-netlify": "2.11.0",
-    "gatsby-plugin-offline": "3.10.1",
+    "gatsby-plugin-manifest": "2.12.1",
+    "gatsby-plugin-mdx": "1.10.1",
+    "gatsby-plugin-netlify": "2.11.1",
+    "gatsby-plugin-offline": "3.10.2",
     "gatsby-plugin-react-helmet": "3.10.0",
     "gatsby-plugin-remove-trailing-slashes": "2.10.0",
     "gatsby-plugin-root-import": "2.0.6",
-    "gatsby-plugin-sharp": "2.14.1",
+    "gatsby-plugin-sharp": "2.14.3",
     "gatsby-plugin-sitemap": "2.12.0",
     "gatsby-plugin-styled-components": "3.10.0",
     "gatsby-plugin-svgr": "2.1.0",
-    "gatsby-plugin-typescript": "2.12.0",
+    "gatsby-plugin-typescript": "2.12.1",
     "gatsby-remark-autolink-headers": "2.11.0",
-    "gatsby-remark-images": "3.11.0",
+    "gatsby-remark-images": "3.11.1",
     "gatsby-remark-smartypants": "2.10.0",
-    "gatsby-source-filesystem": "2.11.0",
-    "gatsby-transformer-sharp": "2.12.0",
+    "gatsby-source-filesystem": "2.11.1",
+    "gatsby-transformer-sharp": "2.12.1",
     "gatsby-transformer-yaml": "2.11.0",
     "husky": "6.0.0",
-    "js-yaml": "4.0.0",
+    "js-yaml": "4.1.0",
     "lint-staged": "10.5.4",
     "lodash": "4.17.21",
     "lodash.debounce": "4.0.8",
-    "markdownlint-cli": "0.26.0",
+    "markdownlint-cli": "0.27.1",
     "polished": "4.1.1",
     "prettier": "2.2.1",
     "prettier-package-json": "2.1.3",
     "prop-types": "15.7.2",
     "react": "16.14.0",
-    "react-beautiful-dnd": "13.0.0",
-    "react-docgen-typescript": "1.21.0",
+    "react-beautiful-dnd": "13.1.0",
+    "react-docgen-typescript": "1.22.0",
     "react-dom": "16.14.0",
-    "react-dropzone": "11.3.1",
+    "react-dropzone": "11.3.2",
     "react-helmet": "6.1.0",
     "react-window": "1.8.6",
     "remark": "13.0.0",
     "remark-react": "8.0.0",
     "slugs": "0.1.3",
     "styled-components": "4.4.1",
-    "stylelint": "13.11.0",
+    "stylelint": "13.12.0",
     "stylelint-config-styled-components": "0.1.1",
     "stylelint-order": "4.1.0",
     "stylelint-processor-styled-components": "1.10.0",
-    "svgo": "2.1.0",
-    "typescript": "4.1.5",
+    "svgo": "2.3.0",
+    "typescript": "4.2.4",
     "unist-util-visit": "2.0.3",
     "vfile": "4.2.1"
   },
   "resolutions": {
-    "@types/vfile-message": "1.0.1"
+    "@types/vfile-message": "1.0.1",
+    "@types/react": "16.14.5"
```

Comment:
@hzhu did you explore other options? These `@types/react` resolutions haven't been very stable for us in the past.

Reply:
You'd have to dig back through PRs across the system to remind yourself. Please `deduplicate` :pray: